iceprog/Makefile

```diff
@@ -10,12 +22,19 @@ CFLAGS += $(shell for pkg in libftdi1 libftdi; do $(PKG_CONFIG) --silence-errors
 
 CFLAGS += -llibusb-1.0
```

Comment:
Wouldn't this look for a liblibusb-1.0? And why is it necessary if we have an `-lusb-1.0` in LDFLAGS?

Reply:
This was an residue from earlier, so cannot answer. But I agree so removed it.